static/css/style.css

```diff
@@ -522,6 +526,68 @@ form.distribute-teams-form, form.clear-teams-form {
     width: 100%;
 }
 
+/* Showcase */
+.carousel.slide {
+    max-width: 800px;
+    margin-left: auto;
```

Comment:
Just a single
`margin: 0 auto;`
is more preferred

Reply:
Changed as suggested.